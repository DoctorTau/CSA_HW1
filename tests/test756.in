39
86 -2760 591 75 7225 7714 -9885 -5662 3691 -9030 4048 -9493 6755 -362 -4012 4751 -4443 6870 1387 7163 604 6510 -8378 -6708 -2028 -9043 -417 -4868 6088 2977 -1279 -2345 6196 3613 -4683 3827 -5463 4774 7282 