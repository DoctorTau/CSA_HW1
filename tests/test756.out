-2760 -5662 -9030 -362 -4012 -8378 -6708 -2028 -4868 